apps/src/templates/teacherHomepage/StudentHomepage.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import RecentCoursesCollapsible from './RecentCoursesCollapsible';
+import CollapsibleSection from './CollapsibleSection';
+import GradientNavCard from './GradientNavCard';
+import shapes from './shapes';
+import i18n from "@cdo/locale";
+
+const StudentHomepage = React.createClass({
+  propTypes: {
+    courses: shapes.courses
+  },
+
+  componentDidMount() {
+    $('#user_hero').appendTo(this.refs.userHero).show();
```

Comment:
Another instance where we would want to protect things.

In this particular instance, would it make more sense to have the user_hero just not be part of React? If it's the first thing on the page I prefer the structure:
```
:haml
  #user_hero
  #container_we_render_react_into